docs/how-to-setup-wsl.md

```diff
@@ -73,10 +75,10 @@ Once you have configured Docker Desktop to work with WSL2, follow these steps to
 
 1. Launch a new Ubuntu-18.04 terminal
 
-2. Pull `MongoDB 4.0.x` from Docker Hub
+2. Pull `MongoDB 4.4.x` from Docker Hub
```

Comment:
We are on MongoDB 6 and will soon move to 7, too. Let's point them to system requirements on the main guide to avoid updating this all the time.